apps/examples/lcd_test/Kconfig

```diff
@@ -9,6 +9,14 @@ config EXAMPLES_LCD
 	---help---
 		Enable LCD Example
 
+if EXAMPLES_LCD
+config EXAMPLE_LCD_FPS_TEST
+	int "LCD FPS test"
```

Comment:
In the menuconfig, this `" "` sentence explains what it is. But I can't get it from the sentence.
```
"the number of frames execution"
```